bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        use_stdin=True,
+        output_format='regex',
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+): (?P<message>.*)')
+class TOMLBear:
+    """
+    This bear validates TOML documents and prints each key's type.
+    """
+    LANGUAGES = {'TOML'}
+    REQUIREMENTS = {GoRequirement('github.com/BurntSushi/toml/cmd/tomlv',
+                                  version='0.1.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return filename
```

Comment:
@Asnelchristian this is my bad... I asked him to remove that comma... I had no idea why :smile: